skimage/io/tests/test_fits.py

```diff
@@ -5,53 +5,50 @@
 from skimage._shared import testing
 
 
-pyfits_available = True
```

Comment:
Can probably be
```
pytest.importorskip("astropy")
```

https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency

And most of `skipif` statements can be ignored.

Reply:
I like this, thanks for pointing it out, @hmaarrfk!